specs/engage-digital_openapi3.yaml

```diff
@@ -6028,6 +6036,14 @@ paths:
             items:
               type: string
             type: array
+        - description: >-
+            Whether the user is passwordless or not (boolean). 

```

Comment:
Nitpick: We don't need 'boolean' here because this information is in the schema. 

Up to you

Reply:
I copied from other properties, I think we can keep the same approach here.